starlette/testclient.py

```diff
@@ -66,16 +66,19 @@ class WebSocketTestSession:
     def __init__(
         self,
         app: ASGI3App,
+        request: httpx.Request,
         scope: Scope,
         portal_factory: _PortalFactoryType,
     ) -> None:
         self.app = app
+        self.request = request
```

Comment:
I think it's too late for some of the other attributes here, but can we make new attributes private?

Reply:
Done, thanks for looking into it :+1: